### PR TITLE
Typos only from the user guide

### PR DESCRIPTION
**Pull request by scoyle391**
This PR addresses actual typos, such as misspelled words, missing words, incorrect word usage, etc.

### CODE REVIEW

**Comment by scoyle391** [comment]
this has an "ugly" link, not a broken one 🙄

File: user-guide/bare-metal.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Install with Bare Metal
 
-In cloud environments, provisioning a readily available network load balancer with Ambassador is the best option for handling ingress into your Kubernetes cluster. When running kubernetes on a bare-metal setup, where network load balancers are not available by default, we need to consider different options for exposing Ambassador. 
+In cloud environments, provisioning a readily available network load balancer with Ambassador is the best option for handling ingress into your Kubernetes cluster. When running Kubernetes on a bare-metal setup, where network load balancers are not available by default, we need to consider different options for exposing Ambassador.
```
Comment (khussey) [inline user-guide/bare-metal.md]: We say "bare-metal" here...

File: user-guide/bare-metal.md [hunk 2]
```diff
@@ -89,4 +89,4 @@ This configuration does not require an Ambassador service be defined so you can
 
 **Note:** Before configuring Ambassador with this method, consider some of the functionality that is lost by bypassing the Kubernetes service including only having one Ambassador able to bind to port 8080 or 8443 per node and losing any load balancing that is typically performed by Kubernetes services. 
 
-Join our [Slack channel](https://d6e.co/slack) to ask any questions you have regarding running Ambassador on a bare metal installation.
+Join our [Slack channel](https://d6e.co/slack) to ask any questions you have regarding running Ambassador on a Bare Metal installation.
```
Comment (khussey) [inline user-guide/bare-metal.md]: But "Bare Metal" here (note it should not be capitalized)..

File: user-guide/cert-manager.md [hunk 3]
```diff
@@ -15,15 +15,15 @@ kubectl create ns cert-manager
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.11.1/cert-manager-no-webhook.yaml
 ```
 
-**Note:** The resource validation webhook is not required and requires addition configuration.
+**Note:** The resource validation webhook is not required and requires additional configuration.
```
Comment (khussey) [inline user-guide/cert-manager.md]: "required and requires" doesn't sound right here...

File: user-guide/monitoring.md [hunk 4]
```diff
@@ -326,7 +326,7 @@ The Prometheus data model is, at its core, time-series based. Therefore, it make
 
 ## Additional Install Options
 
-### Statsd Exporter: Output Statistics to Ambassador Edge Stack
+### D Exporter: Output Statistics to Ambassador Edge Stack
```
Comment (khussey) [inline user-guide/monitoring.md]: Should "D" be "StatsD" here?

File: user-guide/install-ambassador-oss.md [hunk 5]
```diff
@@ -1,5 +1,11 @@
 # The Ambassador API Gateway
 
+**The Ambassador Edge stack is now available and includes additional functionality beyond the current Ambassador API Gateway.** These features including automatic HTTPS, the Edge Policy Console UI, OAuth/OpenID Connect authentication support, integrated rate limiting, a developer portal, and [more](https://www.getambassador.io/edge-stack-faq/).
```
Comment (LukeShu) [inline user-guide/install-ambassador-oss.md]:
```suggestion
**The Ambassador Edge stack is now available and includes additional functionality beyond the current Ambassador API Gateway.** These features including automatic HTTPS, the Edge Policy Console UI, OAuth/OpenID Connect authentication support, integrated rate limiting, a developer portal, and [more](/edge-stack-faq/).
```

**Comment by scoyle391** [comment]
This is blocked by https://github.com/datawire/ambassador-docs/pull/241

**Comment by scoyle391** [comment]
This is also blocked by https://github.com/datawire/getambassador.io/pull/184

**Pull request merged**